pkg/apis/controlplane/types.go

```diff
@@ -82,12 +82,12 @@ type GroupMember struct {
 	Pod *PodReference
 	// ExternalEntity maintains the reference to the ExternalEntity.
 	ExternalEntity *ExternalEntityReference
-	// Node maintains the reference to the Node.
-	Node *NodeReference
 	// IP is the IP address of the Endpoints associated with the GroupMember.
 	IPs []IPAddress
 	// Ports is the list NamedPort of the GroupMember.
 	Ports []NamedPort
+	// Node maintains the reference to the Node.
```

Comment:
change not related to the PR?

Reply:
yes it not related, just found the order was not consistent with type in v1beta2.